.konflux/scripts/subscription-manager-bro.sh

```diff
@@ -12,8 +12,9 @@ SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 
 SECRET_NAME_IN_KONFLUX="subscription-manager-activation-key"
 SECRET_KONFLUX_WORKSPACE_PATH="/workspace/${SECRET_NAME_IN_KONFLUX}"
+SECRET_MOUNT_PATH="/activation-key"
```

Comment:
I'd suggest to add a small comment _what_ mounts the secret in this path. This is buildah task, right?

Reply:
Yes, added comment.